.gitignore

```diff
@@ -111,6 +111,7 @@ ENV/
 /dev.etcd.volumes.json
 /dev.etcd.installed.json
 /env-*.sh
+/dummy-agent-config.toml
```

Comment:
I'd like to rename this to have the namespace first: `agent.dummy.toml`, with dots as the delimeter to indicate the subcomponent.